src/constants.py

```diff
@@ -47,3 +48,14 @@
         "version": "1.9",
     },
 }
+
+SECRETS_UNIT = [
```

Comment:
**suggestion** This seems a bit too low-level for the constants.py module, and indeed it is used only in `core.context`. I would probably put these constants directly there. 

One of the reason why I'm saying this is also that I haven't been able to place very well `constants.py` in the dependency hierarchy yet. Some of the values here seems to be general, and others seems to be product and substrate specific.

This would call for some thoughts I believe which are out of scope here, but in general I would avoid to place here too much content.